projects/components/src/table/data/table-cdk-data-source.ts

```diff
@@ -67,7 +67,7 @@ export class TableCdkDataSource implements DataSource<TableRow> {
     return this.rowsChange$.pipe(
       tap(rows => this.cacheRows(rows)),
       tap(rows => this.cacheFilterableValues(rows)),
-      tap(rows => this.loadingStateSubject.next({ loading$: of(rows), hide: rows.length > 1 }))
+      tap(rows => this.loadingStateSubject.next({ loading$: of(rows), hide: rows.length > 0 }))
```

Comment:
shit happens

Reply:
off by 1! If you can think of a good way to test this, it might be worth it. Nothing obvious comes to mind for me though, since a dom selector won't be able to tell that the row had an element above it blocking clicks.